Linux/Ubuntu-20/Build/manifest.yml

```diff
@@ -0,0 +1,75 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: azure-pipelines-build-agent-ubuntu-20
+spec:
+  replicas: __agentCount__
+  revisionHistoryLimit: 2
+  selector:
+    matchLabels:
+      app: azure-pipelines-build-agent-ubuntu-20
+  template:
+    metadata:
+      labels:
+        app: azure-pipelines-build-agent-ubuntu-20
+    spec:
+      terminationGracePeriodSeconds: 300
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                  - key: agentpool
+                    operator: In
+                    values:
+                      - __agentPoolName__
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+            - weight: 1
+              podAffinityTerm:
+                labelSelector:
+                  matchExpressions:
+                    - key: app
+                      operator: In
+                      values:
+                        - azure-pipelines-build-agent-ubuntu-20
+                topologyKey: failure-domain.beta.kubernetes.io/zone
+      containers:
+        - name: azure-pipelines-build-agent-ubuntu-20
+          image: __PublicAcrName__.azurecr.io/azure-pipelines-build-agent-ubuntu-20:__ImageVersion__

```

Comment:
Maybe now is a good time to hardcode latest?

Reply:
I think handy to have flexibility of the release using build's version.